scripts/patch-specs.sh

```diff
@@ -37,6 +37,9 @@ patch_specs() {
 					mode ~ "link" {
 						sub("%{L.}", "%{L*} -L %:getenv(STAGING_DIR /usr/lib) -rpath-link %:getenv(STAGING_DIR /usr/lib)")
 					}
+					mode ~ "link" {
+						sub("%@{L.}", "%@{L*} -L %:getenv(STAGING_DIR /usr/lib) -rpath-link %:getenv(STAGING_DIR /usr/lib)")
```

Comment:
https://patchwork.ozlabs.org/patch/1098467/

Reply:
huh?

Reply:
Error?

Reply:
I wanted to ask why is this needed ;-)

Reply:
@hauke 

You can try without the patch-specs.sh change and see what happens when compiling packages especially e2fsprogs and iproute2 on x86_64 and may fix in a reasonable way. BTW, you commented on the outdated change.